src/EA.Weee.RequestHandlers/AatfReturn/NonObligated/AddNonObligatedDataAccess.cs

```diff
@@ -23,5 +27,10 @@ public Task Submit(IEnumerable<NonObligatedWeee> nonObligated)
 
             return context.SaveChangesAsync();
         }
+
+        public async Task<List<NonObligatedWeee>> GetNonObligatedByWeee(Guid returnId, bool dcf)
```

Comment:
Add unit test for this... you can create a fake DBset for the context and check that this returns the correct filtered list